kola/tests/packages/ipvsadm.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2017 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package packages
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/coreos/go-semver/semver"
+
+	"github.com/coreos/mantle/kola/cluster"
+	"github.com/coreos/mantle/kola/register"
+)
+
+func init() {
+	register.Register(&register.Test{
+		Run:         ipvsadm,
+		ClusterSize: 1,
+		MinVersion:  semver.Version{Major: 1520},
```

Comment:
This will need to be changed if https://github.com/coreos/coreos-overlay/pull/2707 and https://github.com/coreos/portage-stable/pull/575 don't make it in today